src/mapkit.cs

```diff
@@ -1928,7 +1973,7 @@ interface MKLocalSearchCompleterDelegate {
 
 	[NoWatch]
 	[TV (9,2)][NoWatch][iOS (9,3)]
-	[BaseType (typeof(NSObject))]
+	[BaseType (typeof (NSObject))]
```

Comment:
Please do not change these, it introduces noise.

```suggestion
	[BaseType (typeof(NSObject))]
```